docs/resources/docker-examples/kafka/docker-compose.yaml

```diff
@@ -1,14 +1,12 @@
 version: '2'
 services:
   zookeeper:
-    platform: linux/x86_64
     image: confluentinc/cp-zookeeper:latest
     environment:
       ZOOKEEPER_CLIENT_PORT: 2181
       ZOOKEEPER_TICK_TIME: 2000
 
   kafka:
-    platform: linux/x86_64
     image: confluentinc/cp-kafka:latest
```

Comment:
Why did you remove this? It won't work on a machine that uses a different architecture (it won't work on my Mac M1, for instance).

Reply:
`platform` is only defined after `2.4`. The number is now increased from 2 to 2.4.